lib/matplotlib/text.py

```diff
@@ -2092,7 +2093,9 @@ def draw(self, renderer):
                 self.arrow_patch.figure = self.figure
             self.arrow_patch.draw(renderer)
 
-        Text.draw(self, renderer)
+        astext = copy.copy(self)
+        astext.__class__ = Text
```

Comment:
Evidently it works, but this seems like a hack.  Maybe what it is telling us is that the Annotation could be implemented more cleanly via composition than via inheritance.  That's speculation; I haven't looked closely.